great_expectations/expectations/metrics/map_metric_provider/map_condition_auxilliary_methods.py

```diff
@@ -330,31 +330,29 @@ def _sqlalchemy_map_condition_unexpected_count_value(
 
     try:
         if execution_engine.engine.dialect.name.lower() == GXSqlDialect.MSSQL:
```

Comment:
Should this be updated to `execution_engine.dialect_name`?

Reply:
Good catch! Changing.